torch/_dynamo/config.py

```diff
@@ -103,6 +103,12 @@
 # Show a warning on every graph break
 print_graph_breaks = False
 
+# Show a warning for every shape guard
+print_shape_guards = False
```

Comment:
IMO kill this and gate this code under TORCHDYNAMO_PRINT_GUARDS?

Reply:
I did as @voznesenskym asked for now.